lib/syslog.rb

```diff
@@ -0,0 +1,10 @@
+begin
+  require 'syslog_ext'
```

Comment:
I think you could keep the extension library name `syslog` and load it by `require 'syslog.so'`.  Doesn't it work?